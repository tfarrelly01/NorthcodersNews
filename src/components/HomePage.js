import React from 'react';

const HomePage = () => <h1>Home Page (All Articles)</h1>;

export default HomePage;