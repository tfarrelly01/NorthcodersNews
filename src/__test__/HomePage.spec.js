import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../components/HomePage';

test('HomePage renders correctly', () => {
  const component = shallow(<HomePage />);
  expect(component).toMatchSnapshot();
});