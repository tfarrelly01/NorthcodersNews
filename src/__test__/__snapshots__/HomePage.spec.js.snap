// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders correctly 1`] = `
<h1>
  Home Page (All Articles)
</h1>
`;
